BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/TransactionDocs/TransactionDocsView.swift

```diff
@@ -120,10 +120,16 @@ public class TransactionDocsView: UIView {
                 })
 
                 presentingViewController.present(viewController, animated: true)
-                // TODO: PP-805 Simulate data loading delay
+                // TODO: PP-805 Simulate data loading delay with first result as error
                 DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
                     viewController.stopLoadingIndicatorAnimation()
-                    viewController.setData(viewModel: viewModel)
+                    viewController.setError(errorType: .unexpected, buttonAction: {
+                        viewController.startLoadingIndicatorAnimation()
+                        DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
```

Comment:
[clarification] Do we really need a second delay? Why do we need delays here in general?

Reply:
It will be removed after merging it into 809
In 809 there is already real integration and it doesn't need this code inside TransactionDocsView

Reply:
Understood.